### DISCUSSION TITLE
Can I remove an interface's MTU value with a REST PATCH?

### DISCUSSION THREAD

**Discussion by grantmortenson-experity** [netbox-community/netbox#13762] [Q&A] [answered]
Hello, I am trying to use the REST API to update several interfaces that were improperly configured with an MTU value when they should not have any value assigned. I am struggling to find a way to remove the MTU value with the PATCH method. I tried setting the value to 0, but got an input validation error instead.

Is it possible to clear the MTU value in this way or do I need to use PUT? I am using [NetboxPS](https://github.com/benclaussen/NetboxPS) as a wrapper around the API (I'm new to using REST APIs so this seemed like a good stepping stone to further my learning) and it currently doesn't use the PUT method in any cmdlets.

Any advice you can offer is appreciated. Thanks!

**Comment by candlerb** [comment, marked as answer]
Have you tried PATCH with null as the value?

This is what I see if I query an interface with no MTU set:

```
      "mtu": null,
```

**Reply by grantmortenson-experity** [reply]
Yes, that works. I'll have to see if I can extend the PowerShell module to allow null values. Thanks for helping me figure this out!
